source/request-as-event-emitter.ts

```diff
@@ -68,11 +80,11 @@ export default (options, input?: TransformStream) => {
 		}
 
 		/* istanbul ignore next: electron.net is broken */
-		if (options.useElectronNet && (process.versions as any).electron) {
+		if (options.useElectronNet && (process.versions as ProcessVersionsWithElectron).electron) {
```

Comment:
No point in typing this properly. It's only used once internally here. I think in this case using `any` is ok.